Readme.md

```diff
@@ -3,15 +3,32 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T802173)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+
+# Data Grid for Blazor - How to edit a row on a separate page
+
+This example shows how to create an edit form on a separate page.
+
+![Edit form on a separate page](images/datagrid-with-external-edit-form.gif)
+
+The example shows how to implement the following:

```

Comment:
```suggestion
The example shows how to do the following:

```

Reply:
However, you've already got `This example shows how to`, and now you've got the same again here `The example shows how to` - this is causing confusion